api/animal/animalModel.js

```diff
@@ -0,0 +1,34 @@
+const db = require('../../data/db-config');
+
+const findAll = async () => {
+  return await db('animals');
+};
+
+const findBy = (filter) => {
+  return db('animals').where(filter);
+};
+
+const findById = async (id) => {
+  return db('animals').where({ id }).first().select('*');
+};
+
+const create = async (animal) => {
+  return db('animals').insert(animal).returning('*');
+};
+
+const update = (id, animal) => {
+  return db('animals').where({ id: id }).first().update(animal).returning('*');
```

Comment:
I like how you're also returning everything for that animal id in both the create and update.